tests/insights/test_features.py

```diff
@@ -0,0 +1,182 @@
+from captum.insights.features import (
+    TextFeature,
+    FeatureOutput,
+    EmptyFeature,
+    ImageFeature,
+    GeneralFeature,
+    _convert_figure_base64,
+)
+
+from tests.helpers.basic import BaseTest
+from unittest.mock import patch
+import torch
+from matplotlib.figure import Figure
+
+
+class TestTextFeature(BaseTest):
+    FEATURE_NAME = "question"
+
+    def test_text_feature_returns_text_as_visualization_type(self):
+        feature = TextFeature(self.FEATURE_NAME, None, None, None)
+        self.assertEqual(feature.visualization_type(), "text")
+
+    def test_text_feature_does_not_use_baseline_and_input_transform(self):
+        def mock_transform(*args, **kwargs):
+            raise AssertionError("baseline transform should not have been used")
```

Comment:
I don't think we need to test for this explicitly, since the input and baseline transforms are being used elsewhere.

Reply:
Removed this test